lib/jwtap/access_handler.rb

```diff
@@ -0,0 +1,78 @@
+def bearer(request)
+  authorization = request.headers_in['Authorization']
+  if authorization
+    match = /^Bearer (\S+)$/.match authorization
+    bearer = match[1] if match
+  end
+
+  bearer
+end
+
+def cookie(var)
+  cookies = cookies var
+  cookie = cookies[var.jwtap_cookie_name] if cookies
+  cookie
+end
+
+def cookies(var)
+  Hash[var.http_cookie.split(';').map { |s| s.split('=').map(&:strip) }] if var.http_cookie
```

Comment:
Not sure how likely either issue is:
1.  If a base64 encoding has equals signs at the end then the split on '=' won't get the whole jwt.
2. If any other cookie in the request has an '=' in it then Hash[] can't parse it and will throw an error.